plugins/dashboard/__init__.py

```diff
@@ -999,6 +1003,12 @@ def _get_plotly_config_and_layout(plot_config):
     if plot_config.get("plot_type") == "numeric_histogram":
         layout["bargap"] = 0
         layout["bargroupgap"] = 0
+    if plot_config.get("plot_type") == "categorical_histogram":
+        xaxis = plot_config.get("xaxis", {})
+        layout["xaxis"] = {
+            "type": "category",
```

Comment:
Tells plotly to treat the x data as categories, otherwise `"1"` (str)  is treated as `1` (number).